### PR TITLE
Ad bug creation button in list view

### PR DESCRIPTION
**Pull request by clelland**
This change exposes a "File new bug" button for each feature in the feature list, which links to the "new issue" page on crbug.com. If the feature has not yet launched, and has a launch bug number, then the new bug will be created as a blocker by default. Feature owners will be cc'd if they are defined on the feature.

Partially addresses #297 (Still need a better way to find the correct components to auto-assign the created bug to)

### CODE REVIEW

**Comment by ebidel** [comment]
Can you post a screenshot?

File: static/elements/chromedash-feature.html [hunk 1]
```diff
@@ -381,18 +386,43 @@
...
+    _computeCrbug: function(link) {
+      var crbugNumber = this._computeCrbugNumber(link);
+      if (crbugNumber) {
+        return '#' + crbugNumber;
       }
       return link;
+    },
+    _computeNewBugUrl: function(feature) {
+      var url = 'https://bugs.chromium.org/p/chromium/issues/entry';
+      var params = ["components=Blink"];
```
Comment (ebidel) [inline static/elements/chromedash-feature.html]: single quotes

**Comment by ebidel** [comment]
LGTM. Just the nit

**Comment by clelland** [comment]
screen shot 2016-07-13 at 1 59 17 pm

File: static/elements/chromedash-feature.html (hunk 1 above)
Comment (clelland) [inline static/elements/chromedash-feature.html]: Done.

**Comment by RByers** [comment]
This is great, thanks!

**Pull request merged**